lib/score_recorder.rb

```diff
@@ -0,0 +1,95 @@
+class ScoreRecorder
+  attr_reader :frames
+
+  def initialize(frames = [])
```

Comment:
a reuse of the same name can be a bit confusing, maybe a game class to tie it all together could be good